qiskit_nature/settings.py

```diff
@@ -43,6 +43,7 @@ def dict_aux_operators(self) -> bool:
                 ),
                 stacklevel=3,
             )
+            warnings.filterwarnings("ignore", category=ListAuxOpsDeprecationWarning)
             self._deprecation_shown = True
```

Comment:
Adding this filter should alleviate the need for `_deprecation_shown` in its entirety, right?

Reply:
No, because it will eventually go inside this if statement again if the flag is not set and it will set the filter to default and output it again.